blas/gonum/level2cmplx128.go

```diff
@@ -939,6 +939,227 @@ func (Implementation) Zhpr2(uplo blas.Uplo, n int, alpha complex128, x []complex
 	}
 }
 
+// Ztpmv performs one of the matrix-vector operations
+//  x := A * x    if trans = blas.NoTrans
```

Comment:
I don't think we use the ':=' update operator in the godoc.

Reply:
It is being used at a couple of places. I'll send a PR to use `=` later.

Reply:
SGTM